tests/baselines/reference/privateNameInInExpression.errors.txt

```diff
@@ -0,0 +1,143 @@
+tests/cases/conformance/classes/members/privateNames/privateNameInInExpression.ts(3,27): error TS2805: Static fields with private names can't have initializers when the '--useDefineForClassFields' flag is not specified with a '--target' of 'esnext'. Consider adding the '--useDefineForClassFields' flag.

```

Comment:
Nit: Consider removing this error from the test by changing compiler settings. It is noise here IMO

Reply:
done :)